development/migrations/data_changes.rst

```diff
@@ -96,6 +96,10 @@ if config "allow_avatar_upload" or "allow_avatar_local" is true, update
         array('config.update', array('allow_avatar', 1)),
     )),
 
+.. note::
+
+    Calls to the Migration Tools wrapped in the if conditional will **not** be automatically reverted, as if statements are ignored by the ``revert_data()`` method. You must define your own ``revert_data()`` method to revert any of these data changes if necessary.
```

Comment:
It's actually both if and custom btw. Pretty sure that is already stated below though.